projects/components/src/metric-card/metric-card.component.ts

```diff
@@ -1,33 +1,51 @@
-import { ChangeDetectionStrategy, Component, Input } from '@angular/core';
+import { ChangeDetectionStrategy, Component, EventEmitter, Input, Output } from '@angular/core';
 import { Color } from '@hypertrace/common';
 import { IconSize } from '../icon/icon-size';
+import { MetricCardLayout } from '../public-api';
 import { MetricCardIndicatorType } from './metric-card';
 @Component({
   selector: 'ht-metric-card',
   styleUrls: ['./metric-card.component.scss'],
   changeDetection: ChangeDetectionStrategy.OnPush,
   template: `
-    <div class="metric-card" [style.backgroundColor]="this.backgroundColor" [style.borderColor]="this.borderColor">
-      <div class="indicator-and-title">
-        <div class="indicator">
-          <ng-container [ngSwitch]="this.indicator">
-            <ng-container *ngSwitchCase="'${MetricCardIndicatorType.Dot}'">
-              <div class="dot" [style.backgroundColor]="this.indicatorColor"></div>
+    <div
+      class="metric-card"
+      [ngClass]="{ selected: this.isSelected, clickable: this.clickable }"
```

Comment:
`selected` class doesn't seem to be used anywhere.

Reply:
oh yes, missed to remove it after adding condition in ngStyle. Removed